modules/light-clients/08-wasm/types/types_test.go

```diff
@@ -21,41 +20,40 @@ import (
 
 	simapp "github.com/cosmos/ibc-go/modules/light-clients/08-wasm/testing/simapp"
 	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
-	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
 	"github.com/cosmos/ibc-go/v7/modules/core/exported"
 	ibctesting "github.com/cosmos/ibc-go/v7/testing"
 )
 
 const (
-	tmClientID                    = "07-tendermint-0"
-	grandpaClientID               = "08-wasm-0"
-	codeHash                      = "01234567012345670123456701234567"
-	trustingPeriod  time.Duration = time.Hour * 24 * 7 * 2
-	ubdPeriod       time.Duration = time.Hour * 24 * 7 * 3
-	maxClockDrift   time.Duration = time.Second * 10
+	tmClientID      = "07-tendermint-0"
+	grandpaClientID = "08-wasm-0"
+	codeHash        = "01234567012345670123456701234567"
+	// trustingPeriod  time.Duration = time.Hour * 24 * 7 * 2
+	// ubdPeriod       time.Duration = time.Hour * 24 * 7 * 3
+	// maxClockDrift   time.Duration = time.Second * 10
 )
 
-var (
-	height          = clienttypes.NewHeight(0, 4)
-	newClientHeight = clienttypes.NewHeight(1, 1)
-	upgradePath     = []string{"upgrade", "upgradedIBCState"}
-)
+// var (
+// 	height          = clienttypes.NewHeight(0, 4)
+// 	newClientHeight = clienttypes.NewHeight(1, 1)
+// 	upgradePath     = []string{"upgrade", "upgradedIBCState"}
+// )
 
 type TypesTestSuite struct {
 	testifysuite.Suite
 	coordinator *ibctesting.Coordinator
 	chainA      *ibctesting.TestChain
-	chainB      *ibctesting.TestChain
+	// chainB      *ibctesting.TestChain
 
 	ctx      sdk.Context
 	store    sdk.KVStore
 	codeHash []byte
 	testData map[string]string
 }
 
-func init() {
-	ibctesting.DefaultTestingAppInit = setupTestingApp
-}
+// func init() {
```

Comment:
QQ: was there a failure here? We need this `init` func to set up the correct simapp (one in 08-wasm)

Reply:
you are right, i'll add it back in the next pr. i'm just confused why the tests didn't fail on this pr run 😕